helm/dagster/values.yaml

```diff
@@ -14,6 +14,10 @@ global:
   # dagster subcharts will not create their own service accounts.
   serviceAccountName: ""
 
+  # You can specify a secret name that supplies the broker URL (in a
+  # DAGSTER_CELERY_BROKER_URL key) and backend URL (in a DAGSTER_CELERY_BACKEND_URL key)
```

Comment:
I think these can move to the `generateCeleryConfigSecret` comment.

For this, maybe
> The name for the secret used to pass Celery broker and backend connection urls. This can generally be left as the default, but can be useful if setting `generateCeleryConfigSecret` to false.